app/helpers/braintree_view_helpers.rb

```diff
@@ -0,0 +1,18 @@
+module BraintreeViewHelpers
```

Comment:
Could we rename this into `BraintreeAdminHelpers` or something similar that indicates this helper is used for admin views?